OutOfSchool/OutOfSchool.BusinessLogic/Services/CurrentUserService.cs

```diff
@@ -288,4 +294,44 @@ private async Task<bool> EmployeeWorkshopRights(Guid workshopId)
 
         return isUserRelatedEmployee;
     }
+
+    private Task<bool> ModeratorHasRights(Guid moderatorId)
+    {
+        if (!IsModerator())
+        {
+            return Task.FromResult(false);
+        }
+
+        var result = UserId == moderatorId.ToString();
```

Comment:
as we do not have - the role check should be enough, but when we merge Hlib with his Moderato table - we can check if the user is additionally present in the moderator table.